src/metabase/session/api.clj

```diff
@@ -305,6 +305,15 @@
         (throttle/with-throttling [(login-throttlers :ip-address) (request/ip-address request)]
           (do-login))))))
 
+(api.macros/defendpoint :post "/password_check"
```

Comment:
Should this be `(api.macros/defendpoint :post "/password-check"` ?

Reply:
haha good catch. Thanks